front/src/Phaser/Game/GameScene.ts

```diff
@@ -1235,5 +1236,15 @@ export class GameScene extends ResizableScene implements CenterListener {
         mediaManager.removeTriggerCloseJitsiFrameButton('close-jisi');
     }
 
+    private bannedUser(){
+        this.scene.start(ErrorSceneName, {
```

Comment:
you should call cleanupClosingScene() before changing to errorScene. this means that you won't need the methods stopJitsi and closeCoWebsite after it.